managed-databases/postgresql-and-mysql/api-cli/migrating-endpoints.mdx

```diff
@@ -0,0 +1,64 @@
+---
+meta:
+  title: Migrating Database Instance endpoints via the Scaleway CLI
+  description: This page explains how to migrate Database Instance endpoints via the Scaleway CLI.
+content:
+  h1: Migrating Database Instance endpoints via the Scaleway CLI
+  paragraph: This page explains how to migrate Database Instance endpoints via the Scaleway CLI.
+tags: managed-database postgresql mysql engine version database engine endpoints
+dates:
+  validation: 2023-11-22
+  posted: 2023-11-22
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+You can migrate your endpoints from a Database Instance to another anytime, using the Scaleway CLI.
+
+The following commands are useful when you [upgrade the version of your database engine](/managed-databases/postgresql-and-mysql/how-to/upgrade-version) via the **Upgrade only** option. In this case your endpoint must be migrated manually.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/how-to/create-a-database)
+  - You have installed and set up the [Scaleway Command Line Interface (CLI)](/developer-tools/scaleway-cli/quickstart)
```

Comment:
- You must have jq library installed on your machine. 

- The destination database must not have an already configured endpoint of the same type. 
  - if you want to migrate a `load_balancer` endpoint type the destination instance must not have a `load_balancer` endpoint. You can use the CLI to remove the already existing endpoint on the destination instance 
  - if you want to migrate a `private_network` endpoint type the destination instance must not have a `private_network` endpoint. You can use the CLI to remove the already existing endpoint on the destination instance